packages/components/src/components/tabs/tabs.scss

```diff
@@ -20,7 +20,10 @@
 
         &--top {
             order: 0;
-            border-bottom: 2px solid var(--general-section-1);
+
+            &__border-bottom {
+                border-bottom: 2px solid var(--general-section-1);
+            }
```

Comment:
We can move this 1 level higher, since it is breaking the BEM rule. I suggest: `dc-tabs__list--border-bottom`